packages/react-menu/src/contexts/menuContext.ts

```diff
@@ -1,15 +1,14 @@
 import * as React from 'react';
-import { createContext, useContextSelector, ContextSelector } from '@fluentui/react-context-selector';
+import { createContext, useContextSelector, ContextSelector, Context } from '@fluentui/react-context-selector';
 import { MenuListProps } from '../components/index';
 import { MenuState } from '../components/Menu/index';
 
-const MenuContext = createContext<MenuContextValue>({
+export const MenuContext: Context<MenuContextValue> = createContext<MenuContextValue>({
```

Comment:
`Context<MenuContextValue>` is required because of api-extractor?

Reply:
yup, to avoid `import()`